ksql-rest-app/src/test/java/io/confluent/ksql/rest/integration/RestApiTest.java

```diff
@@ -117,9 +116,8 @@ public void shouldExecuteStreamingQueryWithV1ContentType() {
 
   @Test
   public void shouldExecuteStreamingQueryWithJsonContentType() {
-    final KsqlRequest request = new KsqlRequest(String.format("SELECT * from %s;",
-                                                              PAGE_VIEW_STREAM),
-                                                Collections.emptyMap());
+    final KsqlRequest request = new KsqlRequest(
```

Comment:
Might be worth adding in a test to prove it all hangs together when providing an offset.

Reply:
OK